components/src/utils/Cortex.js

```diff
@@ -62,9 +63,37 @@ export function cortexFetch(input, init) {
               type: '',
               id: '',
             };
+            const props = {};
+
             let debugMessages = '';
+            let debugMessageKey = '';
             for (let i = 0; i < data.messages.length; i++) {
-              debugMessages = debugMessages.concat(`${data.messages[i]['debug-message']} \n `);
+              const message = data.messages[i].data;
+              props.fieldName = message['field-name'];
```

Comment:
Having to manage all these additional props can be time consuming for messages. Let's see if there's an easier way for this.

Reply:
Discussed option on Slack. We can keep the "override" in our properties file as a basic string. If a message does not exist, then we can fall-back to the one returned by Cortex. This eliminates us having to manage every error message.

I'd want to avoid us having to manually map every messages string replacement ourselves as these have the potential to change in the API. The message keys/IDs themselves should not, so we can depend on those.